proof/crefine/X64/Tcb_C.thy

```diff
@@ -4512,39 +4512,104 @@ lemma scast_seL4_TCBFlag_fpuDisabled[simp]:
   "scast seL4_TCBFlag_fpuDisabled = tcbFlagToWord FpuDisabled"
   by (clarsimp simp: seL4_TCBFlag_fpuDisabled_def tcbFlagToWord_def)
 
+lemma scast_seL4_TCBFlag_MASK_numFlags[simp]:
+  "scast seL4_TCBFlag_MASK = mask numFlags"
+  by (clarsimp simp: seL4_TCBFlag_MASK_def numFlags_def)
+
+\<comment> \<open>FIXME: move\<close>
```

Comment:
Please prefer `(* *)` comments for FIXMEs, for consistency.
Also, is this forgotten in this PR or to be done in a future PR?